products/netapp.md

```diff
@@ -0,0 +1,72 @@
+---
+title: NetApp ONTAP
+category: os
+tags: netapp storage-os
+iconSlug: netapp
+permalink: /netapp-ontap
+alternate_urls:
+- /ontap
+- /ontap-os
+versionCommand: system-get-version
+releasePolicyLink: https://mysupport.netapp.com/site/info/version-support
+eolColumn: End of Limited Support
+eoasColumn: End of Full Support 
+eoesColumn: End of Self-Service Support
```

Comment:
```suggestion
eolColumn: Limited Support
eoasColumn: Full Support 
eoesColumn: Self-Service Support
```

So that it works well with what's in the table (such as `Ends in 1 year and 7 months `).